src/handlers/upload.rs

```diff
@@ -391,35 +395,128 @@ pub fn install_forc_at_path(forc_version: &str, forc_path: &Path) -> Result<(),
         }
     };
 
+    for component in missing_components(forc_path, FORC_COMPONENTS) {
```

Comment:
note: the `forc-doc` component is only available in the most recent versions of sway (from what I can recall). If we can foresee `install_binaries_at_path` being called with previous versions of forc (where forc-doc may not have existed) then we may need to add an additional check to not install the component unless minimum supported version is satisfied.

Reply:
done in [32025ea](https://github.com/FuelLabs/forc.pub/pull/90/commits/32025ea540a15868c64ec15a55e5050337c2335b)